site/en/guide/mixed_precision.ipynb

```diff
@@ -531,15 +531,15 @@
       "source": [
         "### Loss scaling background\n",
         "\n",
-        "The basic concept of loss scaling is simple: simply multiply the loss by some large number, say $1024$. We call this number the *loss scale*. This will cause the gradients to scale by $1024$ as well, greatly reducing the chance of underflow. Once the final gradients are computed, divide them by $1024$ to bring them back to their correct values.\n",
+        "The basic concept of loss scaling is simple: simply multiply the loss by some large number, say $1024$. You can call this number the *loss scale*. This will cause the gradients to scale by $1024$ as well, greatly reducing the chance of underflow. Once the final gradients are computed, divide them by $1024$ to bring them back to their correct values.\n",
```

Comment:
This number is called the *loss scale*

Reply:
OK. I understand that the technique for mixed precision training is called "loss scaling" (https://arxiv.org/abs/1910.12385). We can say "a loss scale value" or "a loss scale hyperparameter". And the guide appears to describe the technique ("loss scaling") first. Let's remove the original first person "we" and my suggested "you can" and replace it as follows:

```diff
The basic concept of loss scaling is simple:
- simply multiply the loss by some large number, say $1024$, 
+ multiply the loss by some large number, say $1024$, 
- We call this number the *loss scale*...
- You can call this number the *loss scale*.
+ and you get the *loss scale* value.
...
```

(I've removed the second "simple" to avoid repetition/duplication.)